circuit-types/Cargo.toml

```diff
@@ -36,7 +36,7 @@ renegade-crypto = { path = "../renegade-crypto" }
 byteorder = "1.5"
 itertools = "0.10"
 lazy_static = "1.4"
-serde = { version = "1.0.139", features = ["serde_derive"] }
+serde = { version = "=1.0.197", features = ["serde_derive"] }
```

Comment:
Let's just inherit the workspace version in all these crates

Reply:
ack